### ISSUE TITLE
Can't specify ImagePullSecrets for clickhouse subchart

### ISSUE THREAD

**Issue by dledanseur** [sentry-kubernetes/charts#329]
**Problem:**
When specifying a value for a subchart's imagePullSecrets, helm fails with the following error:
```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(StatefulSet.spec.template.spec.imagePullSecrets[0].name): invalid type for io.k8s.api.core.v1.LocalObjectReference.name: got "array", expected "string"
```
**Steps To reproduce:**

```
helm repo add sentry https://sentry-kubernetes.github.io/charts
helm install sentry sentry/sentry --dry-run --set clickhouse.clickhouse.imagePullSecrets={docker-hub}
```

Same error using values files and without the `--dry-run` parameter.

**Additional information:**

By directly targetting the clickhouse chart, helm generates the manifests correctly:

```
helm install sentry sentry/clickhouse --dry-run --set clickhouse.imagePullSecrets={docker-hub}
```

**Comment by dledanseur** [comment]
Seems already corrected by 573ca29d03bf2c044004c1aa387f652a36ada23a, but the dependency in the sentry chart has not been updated yet. Any plan for that ?

**Comment by Mokto** [comment]
I'll try to do it today.
